src/weatherSource/WeatherSource.js

```diff
@@ -0,0 +1,129 @@
+import {
+    relative
+} from "path";
+
+const API_KEY = '953349aaa2569f9cd4821f8c2ffda23a'
+
+class WeatherAPI {
```

Comment:
Kiedy masz plik i jedną klasę w tymże pliku, klasa powinna nazywać się tak, jak plik - albo w jedną, albo w drugą stronę potrzebne jest tu uzgodnienie.

Druga sprawa: ta klasa służy konkretnie do łączenia z OpenWeatherMap i niczym innym. To samo w sobie jest ok, ale powinno znaleźć odzwierciedlenie w nazwie klasy. Moja propozycja to OpenWeatherMapApiWeatherSource - jasne, jest długie, ale z samej nazwy widać, do czego klasa służy. To ma znaczenie przy utrzymaniu kodu (co nie ma może wielkiego znaczenia w małym dwutygodniowym projekcie edukacyjnym, ale jest dość istotne w projektach, które naprawdę do czegoś służą ;) ).

Reply:
Ok, zmienię to. Co prawda taka nazwa klasy, moim zdaniem, średnio pasuje. Można też zmienić nazwę na samo `OpenWeatherMapApi`, nie potrzebne moim zdaniem jest tutaj `..WeatherSource`, co zwiększa niepotrzebnie długość, a sama nazwa tego api mówi za siebie.

Reply:
Z OpenWeatherApi miałbym wątpliwość o tyle, że to brzmi, jakbyśmy reimplementowali OpenWeatherApi a nie łączyli się  znim :D Ale to w sumie szczegóły już ;)